### PR TITLE
Fix: can not scale application in rollout traits

### PR DESCRIPTION
**Pull request by basuotian**
Signed-off-by: Shuai Tian <tians1@xiaopeng.com>

### Description of your changes

Fix: can not scale application in rollout traits #3150

auto-complete rolloutBatches, enable by annotation

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: pkg/oam/labels.go [hunk 1]
```diff
@@ -179,4 +179,7 @@ const (
 
 	// AnnotationWorkloadName indicates the managed workload's name by trait
 	AnnotationWorkloadName = "trait.oam.dev/workload-name"
+
+	// AnnotationRolloutBatchesAutoComplete is used to enable rollout batches auto complete
+	AnnotationRolloutBatchesAutoComplete = "rollout.trait.oam.dev/rolloutBatchesAutoComplete"
```
Comment (wangyikewxgm) [inline pkg/oam/labels.go:184]: please use `rollout.trait.oam.dev/rollout-batches-auto-complete` as lablel

Comment (basuotian) [reply pkg/oam/labels.go:184]: ok

**Review by wangyikewxgm — APPROVED** [review]
LGTM, thanks.

**Review by wonderflow — CHANGES_REQUESTED** [review]
we don't need to add this annotation, the behavior should be default case

**Pull request closed without merging**